### DISCUSSION TITLE
Repairing Robots?

### DISCUSSION THREAD

**Discussion by CorridorWanterer** [nroutasuo/level13#280] [Ideas]
Could it be possible to repair robots?

**Comment by nroutasuo** [comment]
What would they do?

**Reply by CorridorWanterer** [reply]
Honestly, it would probably be either workers that don't spend food or water but may spend electricity (if that is a resource) or metal/tools, or might also help you in the corridors as customizable recruits (you might be able to fine-tune them for what you need like they might increase scavenging efficiency, or give you aid in battle), or they might increase camp defenses. 
I also feel like they would be a little better than recruits, and you might need workers and blueprints to repair said robots. Robots (because they are a little rare) also might not always be fixed if you attempt to do so, and some robots may be better at what they do than others (like some robots might have more potential than others). I also think if you would work on a robot, there might be a chance that you (or a worker) will exceed in fixing a robot, making it better than other robots. 

Of course, there will also be complexity, as to how complex or difficult it would be to fix a certain robot (if a robot is very high-tech and you simply can't work on it unless you have certain materials or blueprints).

Honestly, there are possibilities with much to work with.

Does that answer your question?

P.S. I think repairing robots and what they would do would be random, now that I think about it because you're not simply making your own robot, but rather repairing a robot that had a certain use, and would then make the repaired robot do the same thing as it was designed to do (I'm saying this because with hiring recruits, it is random on what their stats are, and I would think robots would be similar, except you would find them through scavenging, and would need someone to fix them. I don't want robots to replace recruits).
Sorry for typing so much lol.

**Reply by CorridorWanterer** [reply]
Yeah, and you would need blueprints for this.

**Reply by CorridorWanterer** [reply]
Also, cool update!

**Comment by CorridorWanterer** [comment]
Does that make sense?

**Reply by nroutasuo** [reply]
Yes! It's a bunch of interesting ideas. However I think some of it is very similar to worker robots introduced in the last update (late game). Perhaps some items you find while exploring could be turned into these worker robots as well. Also I think it would be fun to have robot followers in the future but I'll have to think about the balancing and if/how they differ from regular followers.

**Reply by CorridorWanterer** [reply]
Ok. Also there are worker bots???
That makes sense.
